src/test/eo/org/eolang/collections/list-tests.eo

```diff
@@ -588,3 +588,16 @@
       lt. > @
         ^.at 0
         x.at 0
+
+# This code should prints "0 1"
```

Comment:
I think, it will be better to make test with `assert-that`, so we will compare result of dataization for 2 objects

Reply:
@MikhailLipanin the problem just occurred, because of the wrong dataization order. Therefore, I do not see it possible to make this test differently.